hypothesis-python/src/hypothesis/internal/escalation.py

```diff
@@ -102,23 +103,35 @@ def get_trimmed_traceback(exception=None):
     return tb
 
 
-def get_interesting_origin(exception):
+class InterestingOrigin(NamedTuple):
```

Comment:
There's no semantic change here, but with five attributes I wanted them to be named, and at that point a `from_*` constructor seems more idiomatic than an unrelated function.

Reply:
Hmm, nope, blocked by upstream https://github.com/python/mypy/issues/9397 (@sobolevn might know more?)

Function it is, I guess :confused:

Reply:
Nope, sorry. This is the first time I see this problem.
My guess is that there's a diffference between `semanal` (+ decorator plugin) and `semanal_namedtuple`